bird2/files/bird.conf.jinja2

```diff
@@ -131,3 +130,46 @@ protocol bgp {{ iface }}_v6 {
 {%- endfor %}{# ipaddress in interfaces[iface]['ipaddresses'] #}
 {%- endif %}{# iface == uplink_vlan['interface'] #}
 {%- endfor %}{# iface in interfaces #}
+
+{%- if 'nextgen-gateway' in salt['pillar.get']('netbox:role:name') %}
+# If we are a Gateway we announced OSPF default routes
+protocol ospf v2 {
+        area 0 {
+                {%- for iface in salt['pillar.get']('netbox:config_context:network:gateway') %}
+                interface "{{ iface }}" {
+                        type broadcast;		# Detected by default
+                        cost 10;		# Interface metric
+                        hello 5;		# Default hello perid 10 is too long
+                };
+                {%- endfor %}
+        };
+        ipv4 {
+                import none;
+                export filter DEFAULT;
+        };
+}
+{%- elif 'dummy0' not in interfaces or 'dummy1' not in interfaces %}
```

Comment:
This is checking whether we don't have any dummy interface, right? I think this should be
```suggestion
{%- elif 'dummy0' not in interfaces and 'dummy1' not in interfaces %}
```